lib/dependency_spy/cli.rb

```diff
@@ -36,6 +37,8 @@ class CLI < Thor
       DependencySpy::Formatters::Yaml
     ]
 
+    SEVERITY_OPTIONS = ['low', 'medium', 'high']
```

Comment:
This is going to be added to https://github.com/rtfpessoa/yavdb/blob/master/lib/yavdb/constants.rb#L33 in version `0.4.1` that I just released.

Reply:
Done.